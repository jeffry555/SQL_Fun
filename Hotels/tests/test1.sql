
use hotel_db;

call find_lowest_prices( '2022-01-02', '2022-01-05', 1 );

    
