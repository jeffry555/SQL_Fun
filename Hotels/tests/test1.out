name	first_night	last_night	total_cost
Moonlight Hotel	2022-01-03	2022-01-03	100.00
Moonlight Hotel	2022-01-05	2022-01-05	100.00
